src/plugins/charts/public/static/components/collections.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { $Values } from '@kbn/utility-types';
+
+export const ColorModes = Object.freeze({
```

Comment:
I think we can use `enum` here:
```ts
export enum ColorModes {
  BACKGROUND = 'Background',
  LABELS = 'Labels',
  NONE = 'None',
}
```

and for `Rotates` as well.

Reply:
I think @nickofthyme introduced these but I forgot what the advantage over enums was. Could you explain, Nick?

Reply:
The short answer, nothing really.

In elastic-charts (https://github.com/elastic/elastic-charts/pull/266) we only use object enums because of the bloat the standard ts `enums` create in the final transpiled javascript.

The regular enum would produce something like

```ts
enum Pet1 {
  Cat = 'Cat',
  Dog = 'Dog',
}

// transpiles to this
var Pet1;
(function (Pet1) {
    Pet1["Cat"] = "Cat";
    Pet1["Dog"] = "Dog";
})(Pet1 || (Pet1 = {}));
```

where a much cleaner output is 

```ts
const Pet2 = Object.freeze({
  Cat: 'Cat' as 'Cat',
  Dog: 'Dog' as 'Dog',
})
type Pet2 = typeof Pet2.Cat | typeof Pet2.Dog;

// transpiles to this
const Pet2 = Object.freeze({
    Cat: 'Cat',
    Dog: 'Dog',
});
```

Using a `const enum` would be clearer but they are not yet supported in babel (https://github.com/babel/babel/issues/8741)